src/db_options.rs

```diff
@@ -943,6 +943,35 @@ impl Options {
         column_descriptors
     }
 
+    /// Updates DBOptions with values parsed from a string.
+    ///
+    /// https://github.com/facebook/rocksdb/wiki/Option-String-and-Option-Map#option-string
```

Comment:
let's write this as:
```
/// See official [wiki](
/// https://github.com/facebook/rocksdb/wiki/Option-String-and-Option-Map#option-string)
/// for more information.
```

to make rustdoc linter happy

Reply:
thanks